runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/Wave.java

```diff
@@ -25,77 +25,83 @@
 package net.runelite.client.plugins.barbarianassault;
 
 import com.google.common.collect.ImmutableList;
-import java.awt.Color;
 import lombok.Getter;
 import net.runelite.api.Client;
 import net.runelite.api.widgets.Widget;
 import net.runelite.api.widgets.WidgetInfo;
 import net.runelite.client.chat.ChatMessageBuilder;
+import java.awt.*;
 
 @Getter
 class Wave
 {
 	private Client client;
 	private final ImmutableList<WidgetInfo> WIDGETS = ImmutableList.of(
-		WidgetInfo.BA_FAILED_ATTACKER_ATTACKS,
-		WidgetInfo.BA_RUNNERS_PASSED,
-		WidgetInfo.BA_EGGS_COLLECTED,
-		WidgetInfo.BA_HITPOINTS_REPLENISHED,
-		WidgetInfo.BA_WRONG_POISON_PACKS,
-		WidgetInfo.BA_HONOUR_POINTS_REWARD
+			WidgetInfo.BA_FAILED_ATTACKER_ATTACKS,
+			WidgetInfo.BA_RUNNERS_PASSED,
+			WidgetInfo.BA_EGGS_COLLECTED,
+			WidgetInfo.BA_HITPOINTS_REPLENISHED,
+			WidgetInfo.BA_WRONG_POISON_PACKS,
+			WidgetInfo.BA_HONOUR_POINTS_REWARD
 	);
 	private final ImmutableList<WidgetInfo> POINTSWIDGETS = ImmutableList.of(
-//base
-		WidgetInfo.BA_BASE_POINTS,
-//att
-		WidgetInfo.BA_FAILED_ATTACKER_ATTACKS_POINTS,
-		WidgetInfo.BA_RANGERS_KILLED,
-		WidgetInfo.BA_FIGHTERS_KILLED,
-//def
-		WidgetInfo.BA_RUNNERS_PASSED_POINTS,
-		WidgetInfo.BA_RUNNERS_KILLED,
-//coll
-		WidgetInfo.BA_EGGS_COLLECTED_POINTS,
-//heal
-		WidgetInfo.BA_HEALERS_KILLED,
-		WidgetInfo.BA_HITPOINTS_REPLENISHED_POINTS,
-		WidgetInfo.BA_WRONG_POISON_PACKS_POINTS
+			//base
+			WidgetInfo.BA_BASE_POINTS,
+			//att
+			WidgetInfo.BA_FAILED_ATTACKER_ATTACKS_POINTS,
+			WidgetInfo.BA_RANGERS_KILLED,
+			WidgetInfo.BA_FIGHTERS_KILLED,
+			//def
+			WidgetInfo.BA_RUNNERS_PASSED_POINTS,
+			WidgetInfo.BA_RUNNERS_KILLED,
+			//coll
+			WidgetInfo.BA_EGGS_COLLECTED_POINTS,
+			//heal
+			WidgetInfo.BA_HEALERS_KILLED,
+			WidgetInfo.BA_HITPOINTS_REPLENISHED_POINTS,
+			WidgetInfo.BA_WRONG_POISON_PACKS_POINTS
 	);
 	private int[] waveAmounts = new int[6];
 	private int[] allPointsList = new int[10];
 	private int[] wavePoints = new int[6];
 	private int[] otherRolesPointsList = new int[4];
 	private String[] descriptions = {
-		" A: ",
-		"; D: ",
-		"; C: ",
-		"; Vial: ",
-		"; H packs: ",
-		"; Total: "};
+			" A: ",
+			"; D: ",
+			"; C: ",
+			"; Vial: ",
+			"; H packs: ",
+			"; Total: "};
 
 	private String[] otherPointsDescriptions = {
-		" A: ",
-		" D: ",
-		" C: ",
-		" H: "
+			" A: ",
+			" D: ",
+			" C: ",
+			" H: "
 	};
-
 	Wave(Client client)
 	{
 		this.client = client;
 	}
-
 	void setWaveAmounts(int[] amounts)
 	{
-		System.arraycopy(amounts, 0, waveAmounts, 0, amounts.length);
+		for (int i = 0; i < amounts.length; i++)
```

Comment:
`waveAmounts = Arrays.copyOf(amounts, amounts.length());` does the same as this, and looks cleaner